lib/assessment.rb

```diff
@@ -13,28 +13,28 @@ def longest(enum)
   # enumerable supplied (i.e. apply the `+` operator)
   # Assume each entry implements a `+` method
   def sum(enum)
-    # Your code goes here
+    enum.inject { |a, b| a + b }
```

Comment:
Correct. You could also have written
`enum.inject(&:+)` (or, in Ruby 2.0 and above, just `enum.inject(:+)`)
And of course, `reduce` would have worked exactly the same.